### PR TITLE
Misc bootstrap pages and accessibility

### PR DESCRIPTION
**Pull request by danielnaab**

### CODE REVIEW

**Comment by mbucknell** [comment]
If you do the login page, that was ticket pubstwo-1596

File: assets/styles/pubswh/main.scss [hunk 1]
```diff
@@ -128,77 +88,13 @@ li.p1{
...
-}
-
-/*media query*/
-@media screen and (min-width: 768px) {
-
   /* ==========================================================================
      Contact styles
      ========================================================================== */
 
+/*media query*/
```
Comment (mbucknell) [inline assets/styles/pubswh/main.scss]: If these are for the contact page, let's move them to pubs_base.scss.

File: assets/styles/pubswh/pubs_base.scss [hunk 2]
```diff
@@ -167,4 +167,46 @@ $fa-font-path: "./fonts" !default;
   }
 }
 
+h2.otherPages {
+  margin: -5px 0 0 17px;
+}
+
+// Styles each browse page
+.browse-page {
```
Comment (mbucknell) [inline assets/styles/pubswh/pubs_base.scss]: Is this used? I'm not seeing it. If not, please remove.

Comment (danielnaab) [reply assets/styles/pubswh/pubs_base.scss]: Yes, it's used in the `pubswh/browse_*.html` templates

File: assets/styles/pubswh/pubs_base.scss [hunk 3]
```diff
@@ -167,4 +167,46 @@ $fa-font-path: "./fonts" !default;
   }
 }
 
+h2.otherPages {
+  margin: -5px 0 0 17px;
```
[the remark this answers is not in the document]
Comment (danielnaab) [reply assets/styles/pubswh/pubs_base.scss]: It's copied over from `main.scss`... but actually, otherPages looks to always be an `h1`. Don't see any reason to keep it.

File: assets/styles/pubswh/main.scss (hunk 1 above)
Comment (danielnaab) [reply assets/styles/pubswh/main.scss]: I don't actually see any of those classes/ids used anywhere. I'll leave for now and we can handle on cleanup.

File: assets/styles/pubswh/pubs_base.scss [hunk 4]
```diff
@@ -167,4 +167,46 @@ $fa-font-path: "./fonts" !default;
...
+      font-weight: bold;
+      &:hover {
+        text-decoration:underline;
+      }
+    }
+  }
+  ul {
+    @include unstyled-list;
+    li {
+      a {
```
[the remark this answers is not in the document]
Comment (danielnaab) [reply assets/styles/pubswh/pubs_base.scss]: Yes, agreed. I did remove some of the styling, but didn't know how much of it to consider desired. Personally, I'd keep the font size but remove the weight, and maybe add the default bullets back.

**Pull request merged**